templates/frontend/objects/article_details.tpl

```diff
@@ -166,15 +173,18 @@
 								{if $author->getLocalizedAffiliation()}
 									<div class="article-author-affiliation">
 										{$author->getLocalizedAffiliation()|escape}
+										{if $author->getData('rorId')}
+											<a href="{$author->getData('rorId')|escape}">{$rorIdIcon}</a>
+										{/if}
 									</div>
 								{/if}
-								{if $author->getOrcid()}
+								{if $author->getData('orcid')}
 									<div class="orcid">
 										{if $author->getData('orcidAccessToken')}
 											{$orcidIcon}
 										{/if}
-										<a href="{$author->getOrcid()|escape}" target="_blank">
-											{$author->getOrcid()|escape}
+										<a href="{$author->getData('orcid')|escape}" target="_blank">
```

Comment:
Worth pinging @ewhanson to see how this might change with his nearly-complete ORCiD work.

Reply:
I think this is already compliant with ORCID move to the core -- I am not sure, but I think the way orcid is displayed should not change any more, but maybe to double check. If it changes, it will change in the core and then also themes need to be considered (again)...

Reply:
I confirmed with @ewhanson that there aren't further changes planned to ORCID for 3.5 that will impact the templates.